roles/kubevirt-resources/tasks/provision.yaml

```diff
@@ -0,0 +1,35 @@
+- name: Login As Super User
+  command: "oc login -u {{ admin_user }} -p {{ admin_password }}"
+  when: cluster=="openshift"
+        and admin_user is not undefined
+        and admin_password is not undefined
+
+- name: Create {{ namespace }} namespace
+  shell: kubectl create namespace {{ namespace }}
+  ignore_errors : true
```

Comment:
I don't want this to fail if the namespace already exists because it doesn't change the outcome of the playbook.  Some deployments of k8s by default create the kube-system namespace, which would break this task.